tool/tctl/common/workload_identity_command.go

```diff
@@ -518,6 +526,24 @@ func (c *WorkloadIdentityCommand) runOverridesCreate(ctx context.Context, client
 		overrides = append(overrides, certs)
 	}
 
+	// Ensure that the user has not provided the Root CA - we only want them to
+	// provide the intermediates that chain to the root CA. If they provide the
+	// root CA, then workloads will end up needlessly distributing the root CA
+	// to validators.
+	for _, chain := range overrides {
+		for _, cert := range chain {
+			// If the issuer and subject are the same, then this is a
+			// "self-signed" certificate.
+			if cert.Issuer.String() == cert.Subject.String() {
+				slog.WarnContext(
```

Comment:
I'm in two minds about making this more aggressive (e.g an error rather than a logged warning) - a few of our tests rely on being able to specify a root CA within the chain but I figure this is unlikely in "real life". WDYT?